app/views/insights/_contributors.html.erb

```diff
@@ -0,0 +1,105 @@
+<html>
+  <head>
```

Comment:
``<html>``, ``<head>``, ``<body>`` tags are already in application.html.erb
I do not think they should be here.

Reply:
Also file names starting with underscore are reserved for partials.
Please have a look at http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials